common/slot_clock/src/lib.rs

```diff
@@ -137,4 +137,13 @@ pub trait SlotClock: Send + Sync + Sized + Clone {
         slot_clock.set_current_time(freeze_at);
         slot_clock
     }
+
+    /// Returns the delay between the start of the slot and when a request for block components
+    /// missed over gossip in the current slot should be made via RPC.
+    ///
+    /// Currently set equal to `unagg_attestation_production_delay` in order to give our node a
+    /// chance to see attestation for the missing block root, but this may be changed in the future.
+    fn single_lookup_delay(&self) -> Duration {
+        self.unagg_attestation_production_delay()
```

Comment:
I would make an argument for having this lower. Because assuming we didn't get the block components over gossip, we would want to have all components within the attestation deadline to attest ourselves.
2.5 secs or 3 secs maybe?

Reply:
what about        `self.unagg_attestation_production_delay() / 2` ? Is that too low? im not sure we'd have a chance to complete the RPC and process the block in time to attest with 3s

Reply:
I think it is right. Looking at the metrics, seems like the average time for a block to get observed is 2 seconds into the slot. Assuming the blobs are published at the same time as blocks, we can expect it to reach around the 2 second mark.
So I think what you have now is good. Have added metrics for these too, so we'll get to observe on the devnet.